Workbooks/Azure Active Directory/Auth Methods Activity/authmethodsactivity.workbook

```diff
@@ -0,0 +1,528 @@
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 6"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "AuditLogs\r\n| where TimeGenerated > ago(30d)\r\n| where \"{Activity}\" contains ActivityDisplayName\r\n| summarize count() by bin(TimeGenerated, 1d), ActivityDisplayName",
+              "size": 0,
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "barchart",
+              "chartSettings": {
+                "xSettings": {
+                  "numberFormatSettings": {
+                    "unit": 0,
+                    "options": {
+                      "style": "decimal",
+                      "useGrouping": true
+                    }
+                  }
+                },
+                "ySettings": {
+                  "numberFormatSettings": {
+                    "unit": 0,
+                    "options": {
+                      "style": "decimal",
+                      "useGrouping": true
+                    }
+                  }
+                }
+              }
+            },
+            "name": "query - 5"
+          }
+        ]
+      },
+      "customWidth": "50",
+      "name": "group - 8"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "## Failure rate by auth method"
+            },
+            "name": "text - 4"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "SigninLogs\r\n| where TimeGenerated > ago(30d)\r\n| project AuthenticationDetails, Status\r\n| where AuthenticationDetails contains \"authenticationMethod\"\r\n| extend AuthenticationDetails = parse_json(AuthenticationDetails)\r\n| mv-expand AuthenticationDetails\r\n| extend method = case(tostring(AuthenticationDetails.authenticationMethod) == \"PHS\", \"Password\", tostring(AuthenticationDetails.authenticationMethod) == \"PHS, StagedRollout\", \"Password\", tostring(AuthenticationDetails.authenticationMethod) == \"CloudOnlyPassword\", \"Password\", tostring(AuthenticationDetails.authenticationMethod))\r\n| where method != \"\"\r\n| project method, Status\r\n| summarize success = countif(Status.errorCode == 0), failure = countif(Status.errorCode != 0) by method\r\n| project method, failureRate = (failure * 1.0) / ((failure + success) * 1.0)\r\n| sort by failureRate desc\r\n",
```

Comment:
Same with this query. (AuthenticationMethod will be friendly and not CloudOnlyPassword + PHS)